doc/config.md

```diff
@@ -0,0 +1,129 @@
+# Configuration format of qem-bot
+
+Configuration, in another way also called metadata of qem-bot, is held in yaml files with one file per "product". Additionally, there is one special config file called `singlearch.yml` which contains a list of packages that exist only on a single architecture.
+Configuration usually spans over multiple files. And default location excepted from qem-bot is `/etc/openqabot/`
+
+```bash
+/etc/openqabot ❯❯❯ tree                   
+.                          
+├── singlearch.yml               
+├── leap154.yml                   
+├── sles15sp3.yml        
+├── micro53.yml          
+...
+```
+Example of `singlearch.yaml`:
+
+```yaml
+- packageone
+- packagetwo
+- onlyppcle
+```
+
+If an incident has a package from this list, a job is automatically marked as without **aggregate**. That means the Incident can be approved without existing aggregate jobs.
+Other yaml files contain the definition of **product** and data either or both **aggregate** and **incidents**
+
+
+## Structure of a product definition
+
+Contains unique identification of a **product**, common settings and `DISTRI` plus `VERSION` variables for openQA.
+
+Example:
+
+```yaml
+product: leap155
+settings:
+  DISTRI: opensuse
+  VERSION: '15.5'
+  SOMETHING: 'something123'
+  ELSE: 'ELSE123'
+```
+
+* `product` - mandatory and must be unique, no other config file can have the same product. 
+* `settings` - mandatory mapping containing `key` : `value` which are used directly in job schedule. From this the pairs `DISTRI` and `VERSION` are mandatory.
+
+## Aggregate part of the config
+
+Used to schedule multiple Incidents in a product as one BUILD named **YYYYMMDD-{counter from 1}**. Jobs scheduled on this part of the config will have multiple vars `*_TEST_REPO` used to add Incident repositories. 
+
+Example:
+
+```yaml
+ aggregate:
+  FLAVOR: Shiny-New-Flavour
+  archs:
+    - aarch64
+    - s390x
+  test_issues:
+    BASE_TEST_ISSUES: Basesystem:17-SP5
+    OS_TEST_ISSUES: Product-Linux:17-SP5
+    SERVERAPP_TEST_ISSUES: Module-Server-Applications:17-SP5
+    DESKTOP_TEST_ISSUES: Module-Desktop-Applications:17-SP5
+    SDK_TEST_ISSUES: Development-Tools:17-SP5
+    PCM_TEST_ISSUES: Public-Cloud:17-SP5
+  onetime: true
+```
+
+* `aggregate` - self-explanatory, mapping containing all needed settings for the aggregate schedule of a given *product*
+* `FLAVOR` - flavour used by the schedule, can be only once per product, mandatory
+* `archs` - list scheduled architectures, mandatory
+* `test_issues` - mandatory, contains a mapping of `*_TEST_ISSUES`, which are used to decide which Incidents to be scheduled in aggregate jobs. Values are `PRODUCT-IDENTIFICATION:VERSION`. `PRODUCT-IDENTIFICATION` as defined in OBS/IBS. `VERSION` dtto, could use different versions for different `*_TEST_ISSUES`.
```

Comment:
"dtto"?

Reply:
Likely "ditto" - which means "I'm bragging with my knowledge of antiquated Latin words". So my suggestion was to use "similarly" instead.